hedera-node/hedera-token-service/build.gradle.kts

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
-plugins { id("com.hedera.hashgraph.conventions") }
+plugins {
+    id("com.hedera.hashgraph.conventions")
+    `java-test-fixtures`
```

Comment:
Is this additional plugin actually needed?

Reply:
Yes, this service now defines a test fixture for other tests to use `FakeFeeRecordBuilder`